app/assets/stylesheets/style.scss

```diff
@@ -241,6 +243,7 @@ label:hover {
   margin-top: 20px;
   margin-bottom: 35px;
   border-radius: 5px;
+  border: 1px solid #D8D8D8;
```

Comment:
Color literals like `#D8D8D8` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#D8D8D8` should be written as `#d8d8d8`